src/main/kotlin/commands/AnnounceCommand.kt

```diff
@@ -0,0 +1,30 @@
+package commands
+
+import Command
+import ConfigManager.readConfigSafe
+import Send.error
+import UserConfig
+import greedyString
+import arg
+import doesLater
+
+object AnnounceCommand : Command("announce") {
+    init {
+        greedyString("content"){
+            doesLater{context ->
+                val content: String? = context arg "content"
+                if (content.isNullOrEmpty()) {
+                    message.error("You need to put announcement here.")
+                    return@doesLater
+                }
+                Main.client?.servers?.find(message.server?.id ?: run {
```

Comment:
yet **again** you're looking for the server for no reason at all. Replace this with the word server